packages/node_modules/nav-frontend-skjema/src/textarea.js

```diff
@@ -39,28 +39,40 @@ class Textarea extends Component {
         }
     }
 
+    renderTextareaElement(textareaRef, textareaClass, feil, textareaId, other) {
+        return (
+            <textarea
+                ref={(textarea) => {
+                    this.tekstomrade = textarea;
+                    if (textareaRef !== undefined) textareaRef(textarea);
+                }}
+                onChange={this.handleChange}
+                className={inputCls(textareaClass, feil)}
+                type="text"
+                id={textareaId}
+                style={{ height: '30px' }}
+                {...other}
+            />
+        );
+    }
+
     render() {
         const { label, maxLength, textareaClass, id, name, feil, tellerTekst, textareaRef, ...other } = this.props;
         const textareaId = id || name || guid();
         const antallTegn = other.value.length;
+        const textareaEl = this.renderTextareaElement(textareaRef, textareaClass, feil, textareaId, other);
 
         return (
             <div className="skjemaelement textarea__container">
                 <label className="skjemaelement__label" htmlFor={textareaId}>
                     {label}
                 </label>
                 <div className="textarea--medMeta__wrapper">
-                    <textarea
-                        ref={(textarea) => {
-                            this.tekstomrade = textarea;
-                            if (textareaRef !== undefined) textareaRef(textarea);
-                        }}
-                        onChange={this.handleChange}
-                        className={inputCls(textareaClass, feil)}
-                        type="text"
-                        id={textareaId}
-                        style={{ height: '30px' }}
-                        {...other}
+                    <EventThrottler
```

Comment:
Gitt at komponenten har default props, blir det litt vanskelig å se hva denne gjør dersom en kun leser denne fila, men det fungerer jo greit.

Reply:
Jeg hadde nok kanskje løst det på en litt annen måte, men det er jo bare smak og behag :) : 
```
<DOMEventComponent onWindowResize={ this.updateHeight }><textarea...></DOMEventComponent>
``` 
Da kunne du utvidet den med x antall on[...] funksjoner etter hvert som en behovene kom, og kanskje litt mer lesbart. Men det blir samme resultat da (med unntak av en ekstra tag for komponenten).

Reply:
Enig i at det ble litt lite lesbart. Skal få forbedret den delen litt.